### ISSUE TITLE
[Feature] Add "Cards with Hover Effects"

### ISSUE THREAD

**Issue by TBorundia** [Rakesh9100/Beautiify#1235]
### Description

1.Basic Hover Card: A simple card that changes its background color or shadow on hover.
2.Image Zoom Card: A card with an image that zooms in slightly when hovered over.
3.Image Slide Card: A card where the image slides to reveal more content underneath when hovered.
4.Flip Card: A card that flips to reveal additional content on the back side when hovered.
5.3D Tilt Card: A card that tilts in 3D space when hovered over.
6.Overlay Card: A card with an overlay that appears with text or icons when hovered.
7.Border Animation Card: A card with animated borders that change color or thickness on hover.
8.Glow Effect Card: A card that emits a glowing effect around its edges when hovered.
9.Text Reveal Card: A card where hidden text slides in or fades in when hovered over.
10.Button Reveal Card: A card that reveals a hidden button or CTA (Call To Action) when hovered.

### Screenshots

To implement different hover effects, customize the CSS for each card type based on the ideas listed above. Each effect can use different CSS properties like transform, transition, box-shadow, opacity, background-color, and more to achieve the desired hover interaction.

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by TBorundia** [comment]
@Rakesh9100 Sir, please assign this issue to me under GSSoC'24.
Thank You!

**Comment by saurabhsingh720** [comment]
@TBorundia so basically u want to add new component of card hover effects. Right!!

**Comment by TBorundia** [comment]
> @TBorundia so basically u want to add new component of card hover effects. Right!!

@saurabhsingh720 ,
Yes Sir, on hovering cards will give different transition effects.

**Comment by saurabhsingh720** [comment]
@TBorundia ok do it and provide your PR as admin he busy.
Give your PR then he will merge with proper labels

**Comment by Rakesh9100** [comment]
Do you want to add all these components in existing Cards category @TBorundia
